Loop54.Shared/Loop54Client.cs

```diff
@@ -17,6 +17,7 @@ namespace Loop54
     /// </summary>
     public class Loop54Client : ILoop54Client
     {
+        public const string CurrentUserLiteral = "(CurrentUser)";
```

Comment:
I don't think this is a good place for this constant, since it's specific to two request types. Also, "literal" is already communicated by `const`. I'd call it `CurrentUserPlaceholder`. Also, please add an XML doc comment explaining it.

Reply:
My second implementation was to place this constant in `public abstract class GetPopularOrRecentEntitiesRequest : Request`. It can be used as `GetPopularOrRecentEntitiesRequest.CurrentUserPlaceholder`.
Or to create a class for this constant in `GetPopularOrRecentEntitiesRequest.cs` file and provide a reference to it in `ForUserId` xml doc, f.e.         
```
/// <summary>
/// A user ID to retrieve the most common/recent entities for that user or null to retrieve the globally most common/recent entities.
 /// <see cref="CurrentUserPlaceholder"/> can be specified to use the user ID from the User-Id request header.
/// </summary>
```